add_docker_to_matrix.sh

```
@@ -5,7 +5,7 @@ set -e
 shopt -s nullglob
 for package_json in $(find . -name '*.json')
 do
-  jq '.DockerMatrix.ImageNames += [ "fedora40"  ]' ${package_json} > ${package_json}.test
+  jq '.DockerMatrix.ImageNames += [ "fedora44"  ]' ${package_json} > ${package_json}.test
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)